jenkinsfile-runner-base-image/packager-config.yml

```diff
@@ -479,16 +479,11 @@ plugins:
     artifactId: "variant"
     source:
       version: "1.3"
-# Warnings (Jenkins-Version: 2.89.1)
-  - groupId: "org.jvnet.hudson.plugins"
-    artifactId: "warnings"
-    source:
-      version: "5.0.1"
-# Warnings Next Generation (Jenkins-Version: 2.204.4)
+# Warnings Next Generation (Jenkins-Version: 2.204.6)
   - groupId: "io.jenkins.plugins"
     artifactId: "warnings-ng"
     source:
-      version: "8.3.0"
+      version: "8.4.1"
```

Comment:
I think we had problem with this version before right?

Reply:
It was 8.4.0. My hope is that they fixed the issue in 8.4.1. Shall we give it a try?

Reply:
FYI This issue is not yet resolved in 8.4.4